src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -241,10 +241,12 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
               if (!btcAdapter)
                 throw new Error(`No adapter available for ${KnownChainIds.BitcoinMainnet}`)
               const value = assetBalance
+
+              const { account: pubkey } = fromAccountId(accountSpecifier)
```

Comment:
Can we re-use the `account` already declared above? It is pulled from the same `accountSpecifier`.

Then we can just do `chainSpecific: { pubkey: account },` below.

Reply:
done